gallery/tutorials/cov2d_simulation.py

```diff
@@ -83,10 +81,10 @@
     f"Load 3D map and downsample 3D map to desired grids "
     f"of {img_size} x {img_size} x {img_size}."
 )
-infile = mrcfile.open(os.path.join(DATA_DIR, "clean70SRibosome_vol_65p.mrc"))
+vols = Volume.load(DATA_DIR + "/clean70SRibosome_vol_65p.mrc", dtype=dtype)
```

Comment:
Use os.path.join for joining dirs.  Its better practice (and pretty much standard).

One reason is that Windows uses `\` (with some exceptions), and so this would probably break.

Reply:
Ah, okay. Thanks!

Reply:
done